game/game/playercontrol.h

```diff
@@ -75,6 +75,68 @@ class PlayerControl final {
 
     using Action=KeyCodec::Action;
 
+    // Forward-Backward, Right-Left
+    static constexpr size_t NumAxes = 2;
+
+    struct AxisStatus { 
+
+        /// Main direction (e.g. W or Up arrow)
+        std::array<bool, 2> main;
+        
+        /// Reverse direction (e.g. S or Down arrow)
+        std::array<bool, 2> reverse;
+
+        /// Returns the current axis value (scale from -1 to 1)
+        auto modifier() const -> float {
```

Comment:
code: `auto modifier()`
comment: `current axis value`

Maybe `auto value() const` is a better name? Also no need to do so many comment and empty lines - code is quite readable as-is